tests/unit/transport/drivers/test_aiokafka.py

```diff
@@ -787,7 +787,7 @@ def assert_create_worker_consumer(
                 api_version=app.conf.consumer_api_version,
                 client_id=conf.broker_client_id,
                 group_id=conf.id,
-                # group_instance_id=conf.consumer_group_instance_id,
+                group_instance_id=conf.consumer_group_instance_id,
```

Comment:
is this passed in from the application? As long this is passed in we are good here

Reply:
https://github.com/faust-streaming/faust/blob/e91c55fea51f77306da2b2a0e1e3718c34081aa3/faust/types/settings/settings.py#L1106-L1119

Looks like it's been passed in for some time, just never got used.